vscode-react/src/components/aKSessionTableBody.component.tsx

```diff
@@ -70,6 +71,16 @@ export const AKSessionsTableBody = ({
 		hidePopper();
 	};
 
+	const displaySessionDeletePopper = (event: React.MouseEvent<HTMLDivElement>, session: Session) => {
+		if (session.state === SessionState.RUNNING) {
```

Comment:
again, nothing happens ? no user feedback ? 
why do you even allow the button in this case ?

Reply:
There is user feedback as described here: https://github.com/autokitteh/vscode-extension/pull/214#discussion_r1549561541

Reply:
refactored as described above